tests/lib/rules/indent.js

```diff
@@ -718,6 +718,30 @@ ruleTester.run("indent", rule, {
                 "}\n",
             ecmaFeatures: { modules: true, defaultParams: true },
             options: [2]
+        },
+        {
+            code:
+                "const HO = 'bah',\n" +
```

Comment:
Slight nitpick: The word "ho" has a negative connotation in the United States. Can you change it to something else?